src/common/metrics/case_growth.ts

```diff
@@ -0,0 +1,49 @@
+import { COLOR_MAP } from 'common/colors';
+import { Level, LevelInfoMap } from 'common/level';
+
+export const METRIC_NAME = 'Infection growth rate';
+
+const SHORT_DESCRIPTION_LOW = 'Active cases are decreasing';
+const SHORT_DESCRIPTION_MEDIUM = 'Active cases are slowly increasing';
+const SHORT_DESCRIPTION_HIGH = 'Active cases are increasing';
+const SHORT_DESCRIPTION_UNKNOWN = 'Insufficient data to assess';
+
+const LIMIT_LOW = 1;
+const LIMIT_MEDIUM = 1.2;
+const LIMIT_HIGH = Infinity;
+
+const LOW_NAME = 'Reduced';
+const MEDIUM_NAME = 'Moderate';
+const HIGH_NAME = 'Elevated';
```

Comment:
So I think these are supposed to be Low / Medium / High (same for hospitalizations and positive_rate) ?

**On Prod:**
![image](https://user-images.githubusercontent.com/206364/81759075-8c5b4700-9478-11ea-96ec-eb217d369e0f.png)


**With your PR:**
![image](https://user-images.githubusercontent.com/206364/81759030-659d1080-9478-11ea-8292-4f8dffa432bc.png)

Reply:
huh